libs/shared/error/src/index.ts

```diff
@@ -0,0 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export {
+  ContentfulClientError,
+  MultiError,
```

Comment:
I have to export `verror`'s `MultiError` here, because it's used in `doRequest` to pass along potentially multiple `PayPalClientError`s.

Reply:
Not exporting this anymore, since `PayPalClientError` now extends `MultiError`.